apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.mustache

```diff
@@ -14,5 +20,9 @@ $(function() {
 <input type="hidden" id="rte-input-{{editor_uuid}}"
        name="{{name}}" value="{{current_file_contents}}"/>
 <div class="pl-rich-text-editor-container">
-  <div id="rte-{{editor_uuid}}" class="border mathjax_ignore"></div>
+    <div id="rte-{{editor_uuid}}" class="border mathjax_ignore"></div>
+    <div class="pl-rich-text-editor-counter-container
+        {{#counter_disabled}}pl-rich-text-editor-counter-container-disabled{{/counter_disabled}}">
+        <div id="rte-counter-{{editor_uuid}}"></div>
+    </div>
```

Comment:
Instead of hiding it, just don't add it...
```suggestion
    {{^counter_disabled}}
    <div class="pl-rich-text-editor-counter-container">
        <div id="rte-counter-{{editor_uuid}}"></div>
    </div>
    {{/counter_disabled}}
```
This also removes the need for the display: none rule in CSS.